src/app/components/multiselect/multiselect.css

```diff
@@ -42,7 +42,12 @@
     .p-multiselect-token-icon {
         cursor: pointer;
     }
-
+    .p-multiselect-token-label {
```

Comment:
You have to format the code. Try `npm run format` because is not like the other styles